### PR TITLE
Regenerate wrapper with latest kvproto-build

### PR DESCRIPTION
**Pull request by ice1000**
Signed-off-by: ice1000 <ice1000kotlin@foxmail.com>

Changes to the wrapper generator:
+ kvproto-build/pull/2
+ kvproto-build/pull/3

### CODE REVIEW

File: src/prost.rs [hunk 1]
```diff
@@ -2,7 +2,6 @@
 #![allow(missing_docs)]
 #![allow(clippy::all)]
 
-pub mod eraftpb {
-    include!("prost/eraftpb.rs");
-    include!("prost/wrapper_eraftpb.rs");
-}
+pub mod eraftpb{
+include!("prost/eraftpb.rs");include!("prost/wrapper_eraftpb.rs");}
+
```
Comment (nrc) [inline src/prost.rs]: Could you exclude this change please?

Comment (ice1000) [reply src/prost.rs]: Ah I'm so sorry

**Comment by nrc** [comment]
@Hoverbear PTAL

**Comment by ice1000** [comment]
```
   --> src/raft.rs:432:18
    |
432 |             .map(|v| v.get_start_index())
    |                  ^^^^^^^^^^^^^^^^^^^^^^^ help: remove closure as shown: `prost::eraftpb::ConfChange::get_start_index`
    |
```

**Comment by ice1000** [comment]
@nrc @Hoverbear PTALA

**Pull request merged**